app/services/bill-runs/two-part-tariff/submit-remove-bill-run-licence.service.js

```diff
@@ -27,7 +28,9 @@ const ReviewLicenceModel = require('../../../models/review-licence.model.js')
 async function go (billRunId, licenceId, yar) {
   await RemoveReviewDataService.go(billRunId, licenceId)
 
-  const licenceRef = await _flagForSupplementaryBilling(licenceId)
+  await _flagForSupplementaryBilling(licenceId, billRunId)
+
+  const licenceRef = await _fetchLicenceRef(licenceId)
```

Comment:
Can we move this down into the `if (!allLicencesRemoved) {` check?

There is no point fetching it if we are not going to use it. Plus it makes it clearer that this information is needed for the banner message.